src/main/java/com/quizapp/enterprise/events/RoundOverEventListener.java

```diff
@@ -17,7 +17,9 @@
     @EventListener
     @MessageMapping("/chat1")
     public void onApplicationEvent(RoundOverEvent event) {
-        messagingTemplate.convertAndSend("/game1/roundOver/" + event.gameCode , event.leaderboard);
+        StringBuilder roundOverCodeString = new StringBuilder("/game1/roundOver/")
+                .append(event.gameCode);
+        messagingTemplate.convertAndSend(roundOverCodeString.toString(), event.leaderboard);
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/